charts/civil-service/values.preview.template.yaml

```diff
@@ -312,6 +312,7 @@ camunda-bpm:
   java:
     image: hmctsprivatetemp.azurecr.io/camunda/bpm:latest
     disableKeyVaults: true
+    image: hmctsprivatetemp.azurecr.io/camunda/bpm:latest
```

Comment:
Kindly remove this as its already added on 313. Its no more required for this PR